### PR TITLE
Add topic specific callbacks for MQTT broker client

### PR DESCRIPTION
**Pull request by fwege**
Allows registering individual callback functions for each topic subscribed via the MQTT broker client.

Addresses #22

### CODE REVIEW

File: seguro/common/broker.py [hunk 1]
```diff
@@ -86,17 +86,35 @@ def __init__(
...
     def __on_connect(self, client, userdata, flags, rc):
         self.logger.info("Connected with result code %i", rc)
 
     def __on_message(self, client, userdata, msg):
         self.logger.debug("Receive msg: %s - %s", msg.topic, str(msg.payload))
         self.message_queue.put(msg)
 
-    def subscribe(self, topic):
-        """Subscribe client to given topic."""
+    def subscribe(self, topic, callback=""):
```
Comment (stv0g) [inline seguro/common/broker.py:101]: The default argument of `callback `should not be string. I would use `None `here.

File: seguro/common/test_broker.py [hunk 2]
```diff
@@ -1,43 +1,51 @@
...
-    assert broker.message_queue.empty() is True
-
-    # Stop listening for new messages
-    broker.stop_listening()
-
-    # Note that messsages send after listening has stopped,
-    # are not put into message queue
-    broker.publish("mytopic", "Hello again!")
-    assert broker.message_queue.empty() is True
+"""
```
Comment (stv0g) [inline seguro/common/test_broker.py:1]: Can you check why all lines of the tests are detected as changed? Did you modify the line endings maybe?

**Review by stv0g — CHANGES_REQUESTED** [review]
Looks got to me :) I only had small comments

File: seguro/common/broker.py (hunk 1 above)
Comment (fwege) [reply seguro/common/broker.py]: Good catch, fixed in 0bf70b9feb8e42c6e4bb7b87ef8d98a16c408b9d

**Review by stv0g — APPROVED** [review]

**Pull request merged**